docs/report/report.tex

```diff
@@ -189,174 +189,164 @@ \subsection{Deployment Cycle}
 \subsection{Probleme während der Durchführung}
 \colorbox{red}{TODO:}
 
+
+
+
 \section{Projekt Meilensteine}
+
 \subsection{Meilenstein 1}
-\colorbox{red}{TODO:}Kurze Einleitung, von wann bis wann ging
+Der erste Meilenstein begann am 29.10.2020 und ging bis zum 11.11.2020. Für den Anfang des Projektes war hier die Planung, Struktur und das Vorgehen des Projektes der Fokus.
 \subsubsection{Ziel}
-\colorbox{red}{TODO:}was haben wir uns vorgenommen, was war das ziel, was wollten wir schaffen?
+Zum Ende des Meilensteins sollte die Struktur und das Grundgerüst für das Projekt fertig gestellt sein.
+Das umfasste sowohl das Konzept der Software-Architektur, die genutzten Technologien (später Technologie-Stack), einer Anforderungsanalyse mit dem Kunden sowie die Befüllung des Backlogs mit den ersten Tickets.Auch musste sich das Team damit beschäftigen, wie es das Projekt durchführen wollte und welche Management- und Teamorganisations-Methodiken dafür angewendet werden sollten.
+Um ein schnellen und reibungslosen  Start für die Entwicklung zu gewährleisten, sollte das Repository entsprechend vorbereitet werden.
 \subsubsection{Probleme}
-\colorbox{red}{TODO:}welche prrobleme sind aufgetreten?
-
+Die ersten Probleme waren die Auswahl des Technologie-Stacks sowie die Auswahl der benötigten Hardware.
+Auch musste sich das Team das Projekt so entsprechend planen, dass jedes Teammitglied durchgehend sinnvoll arbeiten kann.
 \subsubsection{Lösungen}
-\colorbox{red}{TODO:}Welche Lösungen haben wir gefunden?
-
+Da bereits einige Teammitglieder Erfahrung mit verschiedenen Technologie-Stacks und ähnlichen Problemen hatten, wurde sich auf den Technologie-Stack geeinigt, bei dem die meiste Erfahrung im Team bestand.
+Dieser Bestand aus den oben beschriebenen Technologien.
+Um ein sinnvolles kontinuierliches Arbeiten jedes Teammitglieds zu gewährleisten, einigte sich das Team darauf ein ständig gefülltes Ticket-Pool im Repository zu führen, sodass jedes Teammitglied jederzeit selbstständig eine Aufgabe finden und bearbeiten konnte.
 \subsubsection{Product Increment}
-\colorbox{red}{TODO:}Was ist am Ende dabei rumgekommen?
-
+Insgesamt kam am Ende des Milestones nicht im klassischen Sinne ein Product Increment zustande, vielmehr wurden die Grundbausteine für die Entwicklung und das Teamwork gelegt.
+Es wurde sich auf das bereits erwähnte Technologie-Stack entschieden sowie als Hardware ein Raspberry Pi gewählt.
+Für die Entwicklung wurde ein Mono-Repository mit lerna\footnote{https://lerna.js.org/} aufgesetzt, welches Front- und Backend zusammenfasste.
+Durch die Open Source Natur dieses Projektes wurde eine verständliche und hilfreiche Dokumentation für das Projekt  nötig.
+So wurde auch das Repository mit einer einleitenden Readme, einem \glqqHow to Contribute\grqq Guide und einem gefüllten Backlog versehen.
```

Comment:
```suggestion
So wurde auch das Repository mit einer einleitenden Readme, einem \glqq How to Contribute\grqq{} Guide und einem gefüllten Backlog versehen.
```

\grqq{} fügt nach dem " noch ein Leerzeichen ein 😉

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/409/commits/db7fedecbe3f12f759a2c504c4b9f0e5a7061fee